pkg/networkservice/chains/nsmgr/server.go

```diff
@@ -75,6 +76,7 @@ type nsmgrServer struct {
 
 type serverOptions struct {
 	authorizeServer      networkservice.NetworkServiceServer
+	authMonitorOptions   []authMonitor.Option
```

Comment:
Please do the same as we do for auth networkservice.NetworkServiceServer

Reply:
fixed